Permanent/Helpers/FileHelper.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  FileHelper.swift
+//  Permanent
+//
+//  Created by Adrian Creteanu on 10/11/2020.
+//
+
+import Foundation
+
+class FileHelper {
+    var defaultDirectoryURL: URL?
+    
+    init() {
+        do {
+            self.defaultDirectoryURL = try FileManager.default.url(
+                for: .documentDirectory,
+                in: .userDomainMask,
+                appropriateFor: nil,
+                create: true
+            )
+            
+        } catch {
+            print("Error. Could not get documents directory URL.", error)
```

Comment:
Shouldnt we also show the user if an error occured? Is the feature usable if it fails at this point?

Reply:
Thanks. Will handle this in MainViewController and display an alert.